_posts/2017-03-13-fantas-eel-and-specification-4.md

```diff
@@ -133,12 +133,12 @@ I'm pretty sure you can work out how the `Tuple`'s `concat` method can be rewrit
 I'm sure they're the three words you wanted to hear! Let's imagine you're building some system in which you store customer records that look like this:
 
 ```javascript
-const Customer = daggy.tagged(
+const Customer = daggy.tagged('Customer', [
   'name',             // String
   'favouriteThings',  // [String]
   'registrationDate', // Int -- since epoch
   'hasMadePurchase'   // Bool
-)
+])
```

Comment:
I've tried to minimise noise in this PR so did the least invasive edit. Do edit to reflect your preferred style with regard to the placement of the `[]`.